x/oracle/keeper/keeper_test.go

```diff
@@ -2,106 +2,73 @@ package keeper_test
 
 import (
 	"crypto/rand"
-	"fmt"
 	"math/big"
 	"strings"
 	"testing"
 
-	"github.com/cosmos/cosmos-sdk/baseapp"
-	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
-	"github.com/cosmos/cosmos-sdk/simapp"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
-	"github.com/cosmos/cosmos-sdk/x/staking"
-	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	"github.com/stretchr/testify/suite"
-	"github.com/tendermint/tendermint/crypto/secp256k1"
-	tmrand "github.com/tendermint/tendermint/libs/rand"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmtime "github.com/tendermint/tendermint/types/time"
 
 	persistenceapp "github.com/persistenceOne/persistence-sdk/v2/simapp"
 	"github.com/persistenceOne/persistence-sdk/v2/x/oracle/keeper"
+	"github.com/persistenceOne/persistence-sdk/v2/x/oracle/testutil"
 	"github.com/persistenceOne/persistence-sdk/v2/x/oracle/types"
 )
 
-type IntegrationTestSuite struct {
+const (
+	exchangeRate string = persistenceapp.DisplayDenom
+)
+
+type KeeperTestSuite struct {
 	suite.Suite
 
-	ctx         sdk.Context
-	app         *persistenceapp.SimApp
-	queryClient types.QueryClient
-	msgServer   types.MsgServer
+	valAccAddresses []sdk.AccAddress
```

Comment:
Change it to: `accAddresses`.
`valAccAddresses` is confusing sine we already have the `valAddresses`.

Reply:
Done.